backend/lib/services/projects.js

```diff
@@ -37,9 +37,12 @@ function fromResource ({ metadata, spec = {}, status = {} }) {
       createdBy: fromSubject(createdBy),
       owner: fromSubject(owner),
       description,
-      purpose,
+      purpose
+    },
+    status: {
```

Comment:
The idea behind the mapping of resources has been, to only have `data` and `metadata`. If we want to move back to the resource structure we should not do this silently but in one PR for the hole project.